### DISCUSSION TITLE
SyntaxError: Unexpected token 'export' after copying sources to new folder

### DISCUSSION THREAD

**Discussion by jserve** [quasarframework/quasar#10598] [General - Components / Directives / etc] [answered]
Hello everyone,
when I copy my project into a new folder I get follow problem when I do an 'quasar dev', 'yarn install' went fine without problems, so what I am overlooking here?

<pre>
[d:\tmp\test\sit1\node_modules\@quasar\extras\fontawesome-v5\index.js:3
export const fab500px = 'M103.3 344.3c-6.5-14.2-6.9-18.3 7.4-23.1 25.6-8 8 9.2 43.2 49.2h.3v-93.9c1.2-50.2 44-92.2 97.7-92.2 53.9 0 97.7 43.5 97.7 96.8 0 63.4-60.8 113.2-128.5 93.3-10.5-4.2-2.1-31.7 8.5-28.6 53 0 89.4-10.1 89.4-64.4 0-61-77.1-89.6-116.9-44.6-23.5 26.4-17.6 42.1-17.6 157.6 50.7 31 118.3 22 160.4-20.1 24.8-24.8 38.5-58 38.5-93 0-35.2-13.8-68.2-38.8-93.3-24.8-24.8-57.8-38.5-93.3-38.5s-68.8 13.8-93.5 38.5c-.3.3-16 16.5-21.2 23.9l-.5.6c-3.3 4.7-6.3 9.1-20.1 6.1-6.9-1.7-14.3-5.8-14.3-11.8V20c0-5 3.9-10.5 10.5-10.5h241.3c8.3 0 8.3 11.6 8.3 15.1 0 3.9 0 15.1-8.3 15.1H130.3v132.9h.3c104.2-109.8 282.8-36 282.8 108.9 0 178.1-244.8 220.3-310.1 62.8zm63.3-260.8c-.5 4.2 4.6 24.5 14.6 20.6C306 56.6 384 144.5 390.6 144.5c4.8 0 22.8-15.3 14.3-22.8-93.2-89-234.5-57-238.3-38.2zM393 414.7C283 524.6 94 475.5 61 310.5c0-12.2-30.4-7.4-28.9 3.3 24 173.4 246 256.9 381.6 121.3 6.9-7.8-12.6-28.4-20.7-20.4zM213.6 306.6c0 4 4.3 7.3 5.5 8.5 3 3 6.1 4.4 8.5 4.4 3.8 0 2.6.2 22.3-19.5 19.6 19.3 19.1 19.5 22.3 19.5 5.4 0 18.5-10.4 10.7-18.2L265.6 284l18.2-18.2c6.3-6.8-10.1-21.8-16.2-15.7L249.7 268c-18.6-18.8-18.4-19.5-21.5-19.5-5 0-18 11.7-12.4 17.3L234 284c-18.1 17.9-20.4 19.2-20.4 22.6z|0 0 448 512'
^^^^^^

  SyntaxError: Unexpected token 'export'

  - node:vm:352 Object.compileFunction
    node:vm:352:18

  - loader:1031 wrapSafe
    node:internal/modules/cjs/loader:1031:15

  - loader:1065 Module._compile
    node:internal/modules/cjs/loader:1065:27

  - loader:1153 Object.Module._extensions..js
    node:internal/modules/cjs/loader:1153:10

  - loader:981 Module.load
    node:internal/modules/cjs/loader:981:32

  - loader:822 Function.Module._load
    node:internal/modules/cjs/loader:822:12

  - loader:1005 Module.require
    node:internal/modules/cjs/loader:1005:19

  - helpers:94 require
    node:internal/modules/cjs/helpers:94:18

  - quasar.conf.js:12 Object.<anonymous>
    d:/tmp/test/sit1/quasar.conf.js:12:22

  - loader:1101 Module._compile
    node:internal/modules/cjs/loader:1101:14

]

 App • ⚠️   FAIL  quasar.conf.js has JS errors
</pre>

My 'quasar info'
<pre>
Operating System - Windows_NT(10.0.19043) - win32/x64
NodeJs - 16.7.0

Global packages
  NPM - 7.20.6
  yarn - 1.22.11
  @quasar/cli - 1.2.1
  @quasar/icongenie - Not installed
  cordova - Not installed

Important local packages
  quasar - 2.0.4 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app - 3.1.0 -- Quasar Framework local CLI
  @quasar/extras - 1.10.12 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.2.4 -- The progressive JavaScript framework for buiding modern web UI.
  vue-router - 4.0.11
  vuex - 4.0.2 -- state management for Vue.js
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  @babel/core - 7.15.4 -- Babel compiler core.
  webpack - 5.52.0 -- Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.
  webpack-dev-server - 4.0.0 -- Serves a webpack app. Updates the browser on changes.
  workbox-webpack-plugin - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  typescript - 4.2.2 -- TypeScript is a language for application scale JavaScript development
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*

Networking
  Host - DESKTOP-GU8GMTB
  vendek - 10.8.110.11
  VirtualBox Host-Only Network - 192.168.56.1
  WLAN - 192.168.1.134

(node:14608) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at d:\tmp\test\sit1\node_modules\vuex\package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)
</pre>

**Comment by JeffSpies** [comment]
Start with the simple deletion of .quasar and node_modules from your app directory.

If that doesn't do it, how are you copying the project? Can you link/paste your quasar.conf.js file--at least including line 12, but ideally a bit more.

**Reply by jserve** [reply]
I happens also when I do a git clone from my private repo, on my windows machine I just manually copied with CTRL+C and CTRL+V.
The git project not have the .quasar and node_modules in, I can reproduce the error also on my Mac.

(quasar info of the mac):
<pre>
Operating System - Darwin(20.6.0) - darwin/x64
NodeJs - 16.8.0

Global packages
  NPM - 7.21.0
  yarn - 1.22.11
  @quasar/cli - 1.2.1
  @quasar/icongenie - Not installed
  cordova - Not installed

Important local packages
  quasar - 2.0.4 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app - 3.1.0 -- Quasar Framework local CLI
  @quasar/extras - 1.10.12 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.2.4 -- The progressive JavaScript framework for buiding modern web UI.
  vue-router - 4.0.11
  vuex - 4.0.2 -- state management for Vue.js
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  @babel/core - 7.15.4 -- Babel compiler core.
  webpack - 5.52.0 -- Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.
  webpack-dev-server - 4.0.0 -- Serves a webpack app. Updates the browser on changes.
  workbox-webpack-plugin - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  typescript - 4.2.2 -- TypeScript is a language for application scale JavaScript development
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*

Networking
  Host - localhost
  en0 - 192.168.222.205

(node:33005) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at /Users/jani/tmp/opssitrep-frontend/node_modules/vuex/package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)
</pre>

My quasar.conf.js:
[quasar.conf.js.txt](https://github.com/quasarframework/quasar/files/7125858/quasar.conf.js.txt)

Step-by-step list:
<pre>
git clone [private git link]
cd opssitrep-frontend 
ls -a -l
</pre>

Output:
<pre>
rw-r--r--   1 jani  staff     147  8 Sep 06:20 .editorconfig
-rw-r--r--   1 jani  staff      99  8 Sep 06:20 .eslintignore
-rw-r--r--   1 jani  staff    2201  8 Sep 06:20 .eslintrc.js
drwxr-xr-x  13 jani  staff     416  8 Sep 06:20 .git
-rw-r--r--   1 jani  staff     505  8 Sep 06:20 .gitignore
-rw-r--r--   1 jani  staff     200  8 Sep 06:20 .postcssrc.js
-rw-r--r--   1 jani  staff     435  8 Sep 06:20 README.md
-rw-r--r--   1 jani  staff     244  8 Sep 06:20 babel.config.js
-rw-r--r--   1 jani  staff     565  8 Sep 06:20 jsconfig.json
-rw-r--r--   1 jani  staff    1108  8 Sep 06:20 package.json
drwxr-xr-x   4 jani  staff     128  8 Sep 06:20 public
-rw-r--r--   1 jani  staff    6990  8 Sep 06:20 quasar.conf.js
drwxr-xr-x  12 jani  staff     384  8 Sep 06:20 src
-rw-r--r--   1 jani  staff  304361  8 Sep 06:20 yarn.lock
</pre>

<pre>
yarn
quasar dev
</pre>
--> boom, problem

**Comment by jserve** [comment, marked as answer]
Ok, I just have checked it with a clean 'quasar create' file, this line about fontawesome seems somehow (no idea how) slipped in?

**Reply by JeffSpies** [reply]
Yep, line 12, that's the problem.

**Reply by jserve** [reply]
Yeah, no idea how that slipped in... But it's working now. Thanks for the assist :)
